doc/man1/openssl-req.pod.in

```diff
@@ -205,7 +205,8 @@ See L<openssl-format-options(1)> for details.
 =item B<-keyout> I<filename>
 
 This gives the filename to write any private key to that has been newly created
-or read from B<-key>.
+or read from B<-key>. If you want to write the private key and the B<-key> option
+is provided, you should provide the B<-keyout> option explicitly.
```

Comment:
You also need to update the sentence thereafter, which is:
```
If the B<-keyout> option is not given the filename specified in the
configuration file with the B<default_keyfile> option is used, if present.
```
Better do this before your new sentence, for instance as follows:
```
If neither the B<-keyout> option nor the B<-key> option are given the filename specified
in the configuration file with the B<default_keyfile> option is used, if present.
Thus, if you want to write the private key and the B<-key> option
is provided, you should provide the B<-keyout> option explicitly.
```

Reply:
I've updated the text. @t8m, @DDvO - please reapprove.